api/src/main/java/org/openmrs/module/eptsreports/metadata/HivMetadata.java

```diff
@@ -162,5 +162,17 @@ public ProgramWorkflowState getPatientHasDiedWorkflowState() {
 		    EptsReportConstants.GLOBAL_PROPERTY_ART_PROGRAM_UUID);
 		return getProgramWorkflowState(artProgramUuid, "2", "PATIENT HAS DIED");
 	}
+
```

Comment:
execute mvn clean install before committing the changes in order to have formatted code